### DISCUSSION TITLE
Choosing only one book in Vocabulary Builder

### DISCUSSION THREAD

**Discussion by ichnilatis-gr** [koreader/koreader#11459] [Q&A]
How can I see the words of a single book in the Vocabulary Builder? Do I have to deselect all the other books one by one? This can take a long time. Couldn't there be a better way?

Thanks!

**Comment by poire-z** [comment]
Pinging @weijiuqiao, if you have some thoughts.

**Reply by poire-z** [reply]
Oups, ok, you opened a real github issue #11471 (good! github discussions suck!)

**Comment by eliasiturri** [comment]
This is a bit old, but if you tap and hold on a book title, a menu will open where you can choose to select only that book, select or deselect all, etc. 

(I just saw the commit implementing it came after the question)
